dissect/target/plugins/apps/container/docker.py

```diff
@@ -226,11 +226,15 @@ def containers(self) -> Iterator[DockerContainerRecord]:
     @arg(
         "--raw-messages",
         action="store_true",
+        default=False,
+        required=False,
```

Comment:
Can you explain why this is necessary? This is already implied with `--` and `store_true`.